user-docs/pupil-capture.md

```diff
@@ -424,6 +437,22 @@ The exact time window to consider can be set using the `Gaze History Length` fie
 *   Streaming Surfaces with Pupil Capture - Detected surfaces as well as gaze positions relative to the surface are broadcast under the `surface` topic. Check out [this video](http://youtu.be/qHmfMxGST7A) for a demonstration.
 *   Surface Metrics with Pupil Player - if you have defined surfaces, you can generate surface visibility reports or gaze count per surface. See our [blog post](http://pupil-labs.com/blog/2014/07/0392-player-release.html) for more information.
 
+#### Legacy Markers
+
+The legacy surface system used simple square markers, which are way less robust to detect.
+For all new projects we strongly recommend using Apriltags!
+
+<aside class="warning">
+  <strong>Note</strong> - The legacy marker system had a problem in pupil player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in pupil player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!
```

Comment:
```suggestion
  <strong>Note</strong> - The legacy marker system has a problem in Pupil Player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!
```